sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1059,9 +1059,13 @@ void
 handler::ext_oneapi_graph(command_graph<graph_state::executable>& graph)
 ----
 
-|Invokes the execution of a graph. Only one instance of `graph` may be executing,
-or pending execution, at any time. Concurrent graph execution can be achieved by
-finalizing a graph in modifiable state into multiple graphs in executable state.
+|Invokes the execution of a graph. Only one instance of `graph` may be 
+executing, at any time. If a `graph` is submitted multiple times dependencies
+are automatically added to prevent multiple simultaneous executions of 
+an identical graph. 
```

Comment:
"may be executing" isn't strong enough wording now we're enforcing this.
```suggestion
|Invokes the execution of a graph. Only one instance of `graph` will
execute at any time. If `graph` is submitted multiple times, dependencies
are automatically added by the runtime to prevent concurrent executions of 
an identical graph. 
```